cantools/database/can/formats/arxml.py

```diff
@@ -674,9 +674,14 @@ def _load_system_signal(self, system_signal, decimal, is_float):
             category = self._get_unique_arxml_child(compu_method, 'CATEGORY')
 
             if category is None:
-                raise ValueError(
-                    'CATEGORY in compu method {} does not exist.'.format(
-                        compu_method.find('SHORT-NAME').text))
+                # if no category is specified, we assume that the
+                # physical value of the signal corresponds to its
+                # binary representation.
+                return (minimum,
+                        maximum,
+                        factor,
+                        offset,
+                        choices)
```

Comment:
```unit``` and ```comments``` need also be returned in this tuple

Reply:
oops, right. Fixed, thanks for the catch!